llvm/test/CodeGen/WebAssembly/half-precision.ll

```diff
@@ -103,3 +103,57 @@ define <8 x half> @pmax_intrinsic_v8f16(<8 x half> %a, <8 x half> %b) {
   %v = call <8 x half> @llvm.wasm.pmax.v8f16(<8 x half> %a, <8 x half> %b)
   ret <8 x half> %v
 }
+
+; CHECK-LABEL: compare_oeq_v8f16:
```

Comment:
We have many more combination of `fcmp` that https://github.com/llvm/llvm-project/blob/main/llvm/test/CodeGen/WebAssembly/simd-comparisons.ll tests. Do we assume that the semantics are the same with them?

Reply:
I would assume they'd be the same. I'll add some more tests.